### PR TITLE
Discuss new architecture of services.

### PR DESCRIPTION
**Pull request by srp3rd**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -54,114 +54,61 @@ No Rucio services are running at the remote sites.
...
-We need to keep in mind the available network bandwidth between sites and the extra traffic that brokers add.
-Therefore, message TTL should either be set high enough to not expire before a client can process the message or not set at all.
+The approach we will take solves all of these issues.
 
-We should also note the maintenance of custom broker plug-ins.
-ActiveMQ and Kafka plug-ins are written in Java and RabbitMQ plug-ins are written in Erlang.
-It would probably be a lot easier to find a Java programmer than an Erlang programmer if software features were needed to be added or bug fixes implemented.
+First, the Hermes daemon will be modified to transmit two messages instead of one.
+Hermes will send one message to the ActiveMQ broker, which monitoring services can use.
+Hermes will send the second message is sent to a Kafka Message broker.
```
Comment (ktlim) [inline index.rst]:
```suggestion
Hermes will send the second message to a Kafka Message broker.
```

File: index.rst [hunk 2]
```diff
@@ -54,114 +54,61 @@ No Rucio services are running at the remote sites.
...
-
-Note that topics aren't durable or non-durable; the topic subscriptions can either be durable or non-durable.
+Issues
+------
 
-Message Filtering
------------------
+The Hermes service reads messages from the Rucio event database, converts them to messages, and sends those messages to an ActiveMQ message broker.
+Once consumed by a client, these messages are no longer available.
+Generally, a monitoring service reads these messages for reports on the status of Rucio and, therefore, can not be used by any other client.
```
Comment (ktlim) [inline index.rst]:
```suggestion
Generally, a monitoring service reads these messages for reports on the status of Rucio and, therefore, the messages can not be used by any other client.
```

File: index.rst [hunk 3]
```diff
@@ -54,114 +54,61 @@ No Rucio services are running at the remote sites.
...
-The metadata for this ingestion includes the universal unique identifier (UUID), values for data identifier dimension components, the name of the run collection that "owns" the dataset, and eventually provenance information detailing how the dataset was created.
-This metadata will be obtained from the files themselves, if they are self-describing, such as raw image files, or else from separate JSON or YAML documents (or in the message to the broker itself), with one per file or one per batch.  The exact mechanism for generating this "sidecar" metadata is TBD.
-
+We can trigger this ingestion by receiving Rucio messages, examining the message's contents, and sending a message to a service running at the DF, which will ingest the file into a Butler repository at that site.
+The metadata for this ingestion includes the universal unique identifier (UUID), values for data identifier dimension components, the name of the run collection that "owns" the dataset, and eventually, provenance information detailing how the dataset was created.
```
Comment (ktlim) [inline index.rst]:
```suggestion
The metadata for this ingestion includes the universal unique identifier (UUID), values for data identifier dimension components, the name of the run collection that "owns" the dataset, and, eventually, provenance information detailing how the dataset was created.
```

**Review by ktlim — APPROVED** [review]
Looks great.  Thanks.

File: index.rst (hunk 3 above)
Comment (srp3rd) [reply index.rst]: Fixed

File: index.rst (hunk 2 above)
Comment (srp3rd) [reply index.rst]: Fixed

**Pull request merged**